packages/vite/src/node/plugins/css.ts

```diff
@@ -3177,7 +3177,6 @@ async function compileLightningCSS(
           css = css.replace(dep.placeholder, () => dep.url)
           break
         }
-        deps.add(dep.url)
```

Comment:
This line added non-resolved URLs to `deps`. But `deps` should only include file paths.

That URL was passed to `fileToDevUrl` here.
https://github.com/vitejs/vite/blob/51a5569e66bd7f0de79ac14b9e902d1382ccd0aa/packages/vite/src/node/plugins/css.ts#L1056-L1060
Then, this `fs.readFile` was called for svg files.
https://github.com/vitejs/vite/blob/51a5569e66bd7f0de79ac14b9e902d1382ccd0aa/packages/vite/src/node/plugins/asset.ts#L287

Reply:
For the postcss transformer, we didn't add `url()` referenced assets to `deps`.
https://github.com/vitejs/vite/blob/51a5569e66bd7f0de79ac14b9e902d1382ccd0aa/packages/vite/src/node/plugins/css.ts#L1370-L1374
So for now, I think we can simply remove this `deps.add`. In future, maybe we should add `url()` referenced files to `deps` as well.